wuzz.go

```diff
@@ -668,28 +661,14 @@ func (a *App) PrintBody(g *gocui.Gui) {
 		vrb, _ := g.View(RESPONSE_BODY_VIEW)
 		vrb.Clear()
 
-		responseBody := req.RawResponseBody
-		// pretty-print json
-		ctype, _, err := mime.ParseMediaType(req.ContentType)
-		if err == nil && a.config.General.FormatJSON &&
-			(ctype == CONTENT_TYPES["json"] || strings.HasSuffix(ctype, "+json")) {
-			formatter := jsoncolor.NewFormatter()
-			buf := bytes.NewBuffer(make([]byte, 0, len(req.RawResponseBody)))
-			err := formatter.Format(buf, req.RawResponseBody)
-			if err == nil {
-				responseBody = buf.Bytes()
-			}
-		}
+		responseFormatter := formatter.New(a.config, req.ContentType)
+		vrb.Title = "Response body (F9) " + responseFormatter.Title()
```

Comment:
Could you use https://github.com/asciimoo/wuzz/blob/master/wuzz.go#L191 instead of hard-coding the title prefix?